### PR TITLE
Read functionality

### PR DESCRIPTION
**Pull request by Samanthap803b**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -1,23 +1,67 @@
...
+      sign_in_route,
+      sign_out_route,
+    } = this.props;
     return (
-      <div className="app">
-        <PatientIndex meds={this.state.meds} />
-      </div>
+      <>
+        <Router>
+          {/* <Header {...this.props} /> */}
```
Comment (sjproctor) [inline app/javascript/components/App.js:43]: If you aren't using this code you should remove it. It is a best practice to not have commented out code in the main branch.

File: app/javascript/components/pages/Home.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+describe("When Home renders", () => {
+  const patientsArray = [];
+  it("displays a header", () => {
+    const home = shallow(<Home patientsArray={patientsArray} />);
+    const homeHeading = home.find("h1").text();
+    expect(homeHeading).toEqual("Patients");
+  });
+});
+
+// describe("When home renders", () => {
```
Comment (sjproctor) [inline app/javascript/components/pages/Home.test.js:17]: You should not leave commented out code in your PRs.

File: app/javascript/components/pages/Patientinfo.test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+describe("when Patientinfo  renders", () => {
+  it("display patient first name ", () => {
+    const home = shallow(<PatientInfo  />);
+    const homeHeading = home.find("h2").text();
+     expect(homeHeading).toEqual("Stefani");
+  });
+});
+
+// img file
```
Comment (sjproctor) [inline app/javascript/components/pages/Patientinfo.test.js:17]: Make sure these comments get removed. This looks like content that could be in a notes or readme file.

**Review by sjproctor — CHANGES_REQUESTED** [review]
Few things to clean up!

File: app/javascript/components/App.js (hunk 1 above)
Comment (Samanthap803b) [reply app/javascript/components/App.js:43]: Sarah this is for later.

File: app/javascript/components/pages/Home.test.js (hunk 2 above)
Comment (Samanthap803b) [reply app/javascript/components/pages/Home.test.js]: deleted

**Pull request merged**